.textlintrc.json

```diff
@@ -83,10 +83,11 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
```

Comment:
Wait, this rule is wrong(it should use `\\.`) but in any case, it should be merged with the original "dotnet" rule.